src/Airship/Types.hs

```diff
@@ -101,19 +104,21 @@ data ResponseBody
 escapedResponse :: Text -> ResponseBody
 escapedResponse = ResponseBuilder . fromHtmlEscapedText
 
-data Response = Response { _responseStatus     :: Status
-                         , _responseHeaders    :: ResponseHeaders
-                         , _responseBody       :: ResponseBody
+data Response = Response { _responseStatus  :: Status
+                         , _responseHeaders :: ResponseHeaders
+                         , _responseBody    :: ResponseBody
                          }
 
-data ResponseState = ResponseState { stateHeaders   :: ResponseHeaders
-                                   , stateBody      :: ResponseBody
-                                   , _params        :: HashMap Text Text
-                                   , _dispatchPath  :: [Text]
+data ResponseState = ResponseState { stateHeaders  :: ResponseHeaders
```

Comment:
While you're here, mind fixing the inconsistent underscore convention we have in this file? I think we added `_params` and `_dispatchPath` after we dropped in microlens, so now that we have lenses let's just use them everywhere.